CONTRIBUTING.md

```diff
@@ -36,6 +36,13 @@ You can also have `yarn` watch for changes and automatically recompile them:
 yarn watch
 ```
 
+#### Running E2E tests
+
+1. Install [K6](https://k6.io/docs/get-started/installation/)
+1. Ensure Grafana is running on port `:3000`
+1. Ensure clickhouse server is running on port `:9000`, hint you can use the `docker-compose.yml` file to get the needed services running.
+1. Run `k6 run e2e/e2ek6.test.js`
```

Comment:
Let's add it as an npm script, which is a standard way in the JS community. to distribute built-in scripts

Reply:
done.